src/Utils/portals.tsx

```diff
@@ -58,8 +59,17 @@ const getNotificationContainer = (
 export const usePortal = () => {
   const portal = getPortalElement()
 
-  const addElementToPortal = (element: ReactNode): ReactPortal => {
-    return createPortal(element, portal)
+  const addElementToPortal = (
+    element: ReactNode,
+    forceToTop?: boolean
+  ): ReactPortal => {
+    let registeredElement = element
+
+    if (forceToTop) {
+      registeredElement = <div style={{zIndex: 9999}}>{element}</div>
```

Comment:
Wrapping it in a div conditionally might cause some unexpected behavior, especially if the element is used inline, or if a predictable order of elements is being used by some third-party applications like Google Analytics.